docker-compose-dev.yml

```diff
@@ -84,10 +84,16 @@ services:
       - SITE_HOST=http://gateway:8000
 #      - SETTINGS_AUTH_MECHANISM=default #custom_token
 #      - SETTINGS_TOKEN_AUTH_URL=<URL_FOR_TOKEN_VERIFICATION>
```

Comment:
Just as a last comment @akihikokuroda I don't know if we can remove these lines commented if we are not going to need them.

Reply:
These lines were added in https://github.com/Qiskit-Extensions/quantum-serverless/pull/524.  I'll leave them there.